src/yz_index.erl

```diff
@@ -25,6 +25,10 @@
 -include("yokozuna.hrl").
 -compile(export_all).
 
+-define(INITFAILURES(I, S), kvc:path(erlang:iolist_to_binary(
```

Comment:
super nitpicky, but I'd suggest renaming this to EXTRACT_INITFAILURES or SOLR_INITFAILURES just to make the usage a little more obvious